### PR TITLE
[Edit] HTML Add Media: Headings

### PR DESCRIPTION
**Pull request by OhWilko**
### Description

Added content to headings.md covering the topic of changing heading sizes.
closes issue #3229 

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/html/concepts/headings/headings.md [hunk 1]
```diff
@@ -30,6 +30,14 @@ Among the HTML elements are the 6 heading elements. They are labeled 1-6, with 1
 <h5>Heading 5</h5>
 <h6>Heading 6</h6>
 
+## Changing Heading size
+
+HTML Headings have default sizes but these can be changed using the CSS style attribute. Using the font-size property you can customize how big or small headings will appear on the webpage. This is useful if the
+default size does not fill the available space correctly.
```
Comment (dakshdeepHERE) [inline content/html/concepts/headings/headings.md:36]:
```suggestion
HTML Headings have default sizes but these can be changed using the CSS style attribute. Using the font-size property you can customize how big or small headings will appear on the webpage. This is useful if the default size does not fill the available space correctly. 
```

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]
One change from my side

**Review by dakshdeepHERE — APPROVED** [review]
LGTM 🚀

**Review by SSwiniarski — APPROVED** [review]
@OhWilko I made a few changes, and now it's ready to merge.

**Pull request merged**